tests/checkasm/vvc_deblock.c

```diff
@@ -64,11 +66,13 @@ static const uint32_t pixel_mask[3] = {0xffffffff, 0x03ff03ff, 0x0fff0fff};
 #define RANDCLIP(x, diff) av_clip(GET(x) - (diff), 0, \
     (1 << (bit_depth)) - 1) + rnd() % FFMAX(2 * (diff), 1)
 
-static void randomize_params(int32_t beta[4], int32_t tc[4], const int size)
+static void randomize_params(int32_t beta[4], int32_t tc[4], const int is_luma, const int bit_depth, const int size)
 {
+    const int tc_min   = 1 << FFMAX(0, 9 - bit_depth);
+    const int beta_min = is_luma;                     // for luma, beta == 0 will disable the deblock, for chroma beta == 0 is a valid value
     for (int i = 0; i < size; i++) {
-        beta[i] = rand() % 89;
-        tc[i] = rand() % 396;
+        beta[i] = rand() % (89 - beta_min) + beta_min;
+        tc[i] = (rand() % (396 - tc_min)) + tc_min;   // minimum useful value is tc_min, full range [0, 395]
```

Comment:
`rand()` should be `rnd()`

Would this be easier?
```
beta[i] = FFMAX(rnd() % 89, beta_min);
tc[i] = FFMAX(rnd() % 396, tc_min);
```

Reply:
> rand() should be rnd()

sorry for missing this

> Would this be easier?

good idea, fixed